### DISCUSSION TITLE
What is your development workflow?

### DISCUSSION THREAD

**Discussion by pajot** [openego/oeplatform#976] [Q&A]
On 18 May 2022 I conducted a workshop using a prototype environment based on the OEP.

Within minutes we had uncovered issues with login UX; later, there were problems with table creation, data upload and metadata entry (generally error messages and failures).

It seems clear that a lot of work needs to be done to make this usable in production, so the development workflow must be as efficient as possible.

How is your development environment configured? What IDEs do you use? What is your workflow? How do you test your changes? How long does an edit-test cycle typically take?

**Comment by Ludee** [comment]
That's a lot of questions on top of the workflow.
The current version of the development process is documented in [CONTRIBUTING.md](https://github.com/OpenEnergyPlatform/oeplatform/blob/develop/CONTRIBUTING.md).
